shared/services/rewards/types.go

```diff
@@ -143,6 +143,7 @@ type IntervalInfo struct {
 	CID                    string        `json:"cid"`
 	StartTime              time.Time     `json:"startTime"`
 	EndTime                time.Time     `json:"endTime"`
+	TotalNodeWeight        *QuotedBigInt `json:"-"`
```

Comment:
move this below `MerkleProof` and separate it with a newline (ie, keep the serailized fields in a section that the top, drop this to the bottom into a new section)

Reply:
Done